specification/elasticsan/resource-manager/Microsoft.ElasticSan/stable/2023-01-01/elasticsan.json

```diff
@@ -1185,7 +1505,7 @@
         "systemData": {
           "description": "Resource metadata required by ARM RPC",
```

Comment:
You won't need to declare this if you inherit TrackedResource from common-types/v3 or newer